### DISCUSSION TITLE
TMDB大量删除中文标题【还有这事？】谁知道什么情况

### DISCUSSION THREAD

**Discussion by Whichbfj28** [jxxghp/MoviePilot#1682] [讨论]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.7.1

### 运行环境

Docker

### 问题类型

主程序运行问题

### 问题描述

![image](https://github.com/jxxghp/MoviePilot/assets/47915665/6d79a4a8-6743-429c-8daa-8764f44c91b0)
从更更新日志才看到文章.这是为什么？谁知道什么情况

**Comment by copurxia** [comment]
#1653
